src/robots/upper_body.cpp

```diff
@@ -254,6 +293,49 @@ class UpperBody: public cardsflow::kindyn::Robot{
 
     }
 
+    /**
+     * Publish Target point to bullet
+     * @param body_part
+     * @param zeroes_or_current will publish either "zeroes" or "current" as targets to Bullet
+     */
+    void publishBulletTarget(string body_part, string zeroes_or_current){
```

Comment:
see previous comment - avoid strings due to easy typos occurrence

Reply:
Resolved in `fix_L_matrix` branch